examples/step-with-tasks.yaml

```diff
@@ -0,0 +1,57 @@
+---
+- step:
+    name: run training
+    image: tensorflow/tensorflow
+    command: python -u /tmp/code/my_mnist.py
+    inputs:
+      - name: training-set-images
+        default: http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz
+        description: the image data for the training set
+    parameters:
+      - name: B
+        type: integer
+        default: 1
+
+- task:
```

Comment:
I think this example should include every (relevant) type of variant parameter we have:

> SingleVariant
> MultipleVariant
> LinearVariant
> LogspaceVariant
> RandomVariant

(These are names of the classes not values for style)